OrderList.js

```diff
@@ -19,19 +19,16 @@ export class OrderList {
       fetch(
         "https://my-json-server.typicode.com/Solnick/fake-orders-db/products",
       ),
-    ])
-      .then((responsesArray) =>
-        Promise.all(responsesArray.map((res) => res.json())),
-      )
-      .then((data) => (this.fetchedData = data))
-      .then((dataArray) => console.log(dataArray));
-    this.proceedData();
+    ]);
+    this.fetchedData = await Promise.all(responsesArray.map((res) => res.json()));
+    this.transformData();
   }
 
   findItemById(list, id) {
     return list.find((obj) => obj.id === id).name;
   }
-  proceedData() {
+
+  transformData() {
```

Comment:
please don't use magic numbers like `1`.

Because you stored the data in the form of 3 elements array you need to use it this way. Please store the `buyers`, `products` and `orders` in the separate class properties, so when accessing the data we can deduce from the code what data you are accessing

Reply:
So ultimetly, make sure that this function is readable and is not using magic numbers